libraries/plugins/peerplays_sidechain/sidechain_net_handler_bitcoin.cpp

```diff
@@ -1887,7 +1924,23 @@ std::string sidechain_net_handler_bitcoin::sign_transaction_standalone(const sid
 
    std::string pubkey = plugin.get_current_son_object().sidechain_public_keys.at(sidechain);
    std::string prvkey = get_private_key(pubkey);
-   std::string tx = sto.transaction;
+   using namespace libbitcoin;
+   using namespace libbitcoin::machine;
+   using namespace libbitcoin::wallet;
+
+
+   libbitcoin::data_chunk data;
+   std::vector<uint64_t> in_amounts;
+   read_tx_data_from_string(sto.transaction, data, in_amounts);
```

Comment:
Why not using libbitcoin functions?

Reply:
This function serializes not only transaction itself but input amounts info too that is not part of the transaction information. It is our custom format for "transaction + inputs amounts", there is no libbitcoin function for this.